packages/sdk-core/src/model/report/BacktraceReport.ts

```diff
@@ -47,23 +57,29 @@ export class BacktraceReport {
         if (data instanceof Error) {
             this.annotations['error'] = data;
             this.classifiers = [data.name];
-            this.message = data.message;
-            this.stackTrace = data.stack ?? '';
+            const stackInfo = {
+                stack: data.stack ?? '',
+                message: data.message,
+            };
+            this.stackTrace['main'] = stackInfo;
 
             // Supported in ES2022
             if ((data as { cause?: unknown }).cause) {
                 this.innerReport.push((data as { cause?: unknown }).cause);
             }
         } else {
-            this.message = data;
-            this.stackTrace = new Error().stack ?? '';
+            const stackInfo = {
+                stack: new Error().stack ?? '',
+                message: data,
+            };
+            this.stackTrace['main'] = stackInfo;
             errorType = 'Message';
             this.skipFrames += 1;
         }
 
         if (!this.attributes['error.type']) {
             this.attributes['error.type'] = errorType;
         }
-        this.attributes['error.message'] = this.message;
+        this.attributes['error.message'] = this.stackTrace['main']?.message ?? '';
```

Comment:
We still need to set the error message. It's important to have an easy way to verify the report message by the user. Can we still maintain the message property and use it here?

Reply:
I added message back as a field in BacktraceReport. I set it as well as the message in the main stack trace